bin/workbench-daemon.sh

```diff
@@ -140,6 +160,10 @@ function find_workbench_process() {
   fi
 }
 
+
+if [[ "$2" = "getMysqlJar" ]]; then
```

Comment:
Do not increase the parameters of `workbench-daemon.sh`,
When the workbench-daemon is running, Just check if the mysql.jar file is included.
If there is no prompt for the user to enter the [N] key to install itself, `workbench-daemon.sh` exits.
Or enter the [Y] button to download directly.
Because, without mysql.jar, the workbench server is not working.